frontend/src/app/utils/testResults.ts

```diff
@@ -56,6 +56,20 @@ export function hasCovidResults(results: MultiplexResults): boolean {
   );
 }
 
+export function hasDiseaseSpecificResults(
+  results: MultiplexResults | null,
+  diseaseName: MultiplexDisease
+): boolean {
+  if (results) {
+    return (
+      results.filter((multiplexResult: MultiplexResult) =>
+        multiplexResult.disease.name.includes(diseaseName)
+      ).length > 0
```

Comment:
I think we could simplify this check to something like 

```typescript
return results && results
  .map(result => result.disease.name)
  .includes(diseaseName)
```

using `includes` as final truthiness check feels a bit more idiomatic over filtering / checking for length > 0

Reply:
Got it -- thanks Bob!

re: same pattern for the other disease checks -- Yes, I noticed that too. Wanted to minimize scope of these changes so left cleaning that up for later!